docs/grammar/SolidityLexer.g4

```diff
@@ -305,7 +305,7 @@ YulEVMBuiltin:
 	| 'delegatecall' | 'staticcall' | 'return' | 'revert' | 'selfdestruct' | 'invalid'
 	| 'log0' | 'log1' | 'log2' | 'log3' | 'log4' | 'chainid' | 'origin' | 'gasprice'
 	| 'blockhash' | 'coinbase' | 'timestamp' | 'number' | 'difficulty' | 'prevrandao'
-	| 'gaslimit' | 'basefee' | 'blobbasefee';
+	| 'gaslimit' | 'basefee' | 'blobhash' | 'blobbasefee';
```

Comment:
These are not sorted alphabetically so I'd put it next to `blockhash`.

Reply:
Oh, I see that it's between `basefee` and `blobbasefee` in the opcode numbering. I'd still put it next to `blockhash` where possible though, because they're similar in many ways and it makes reviewing easier when they're next to each other.

Reply:
Yes, I tried to follow the opcode numbering order. Ok, I will move it close to `blockhash` then.